WindowsUpdates/WindowsUpdates.psm1

```diff
@@ -93,7 +93,9 @@ function Get-WindowsUpdate {
     [CmdletBinding()]
     Param(
         [Parameter(Mandatory=$false)]
```

Comment:
Change the commit message to:

Added `ExcludeHardwareId` parameter for `Get-WindowsUpdate`

Reply:
Add a Readme section for the new parameter like this section here: https://github.com/cloudbase/WindowsUpdateCLI#if-you-want-to-exclude-kbids